src/cdk/v2/destinations/ortto/procWorkflow.yaml

```diff
@@ -46,7 +47,7 @@ steps:
           "dtz::b": $.getBirthdayObj({{{{$.getGenericPaths("birthday")}}}}),
           "str::ei": {{{{$.getGenericPaths("userId")}}}},
           "str::language": .context.traits.language || .context.locale,
-          "phn::phone": {"n": {{{{$.getGenericPaths("phone")}}}}},
+          "phn::phone": phone ? {"n": phone} : undefined,
```

Comment:
you can skip `: undefined` and write just `phone ? {"n": phone}`

Reply:
Done